mspray/settings.py

```diff
@@ -271,6 +273,8 @@ def skip_suspicious_operations(record):
 ONA_URI = "https://api.ona.io"
 MSPRAY_WEEKLY_DASHBOARD_UPDATE_URL = "http://example.com"
 MSPRAY_AWS_PATH = "mspray"
+MSPRAY_COMMUNITY_HEALTH_WORKER = "community health worker"
+MSPRAY_SUPERVISOR = "supervisor"
```

Comment:
the default should be "Team Leader Assistant (TLA)", which is what it was before now